satpy/scene.py

```diff
@@ -727,8 +729,10 @@ def resample(self,
 
         if destination is None:
             destination = self.max_area(to_resample)
-        new_scn = self._resampled_scene(to_resample, destination,
-                                        **resample_kwargs)
+        new_scn, resamplers, destination_area = self._resampled_scene(to_resample, destination,
+                                                                      **resample_kwargs)
+        for source, (key, resampler) in resamplers.items():
+            self.resamplers[key] = resampler
```

Comment:
Ok I see now. You can't assign to `self.resamplers` in `_resampled_scene` because it is a class method. Is there a reason it has to be a class method? Add this for loop to that method would make sense if it wasn't a classmethod. I could see the class method being useful if it was public or if it was a utility function that users may want to use outside of a Scene, but it isn't right now.

Reply:
This should be removed now right?